### PR TITLE
Add guiding check lists...

### PR DESCRIPTION
**Pull request by ravage84**
... for adding new rule and rule property

This way, we can guide people in providing even more complete PRs.

Type:  documentation improvment
Issue: Resolves none
Breaking change: no

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,4 +11,26 @@ Please check this points before submitting your PR.
...
+  
+ ## Adding a New Rule
+
+- Add the new rule to the matching rule set XML, e.g. ``src/main/resources/rulesets/naming.xml``
+- Add documentation for the new rule, e.g. ``src/site/rst/rules/naming.rst``
+- Implement the new rule, e.g. ``src/main/php/PHPMD/Rule/Naming/LongVariable.php``
+- Cover cases for the new rule in the rule test, e.g. ``src/test/php/PHPMD/Rule/Naming/LongVariableTest.php``
+-- Cover the case when the new rule *should* apply
+-- Cover the case when the new rule *should not* apply
+-- Cover edge cases of th new rule
```
Comment (tvbeek) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: missing the e for the

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -11,4 +11,26 @@ Please check this points before submitting your PR.
...
+
+- Add the new property to rule set XML, e.g. ``src/main/resources/rulesets/naming.xml``
+- Add documentation for the new property, e.g. ``src/site/rst/rules/naming.rst``
+- Implement new property in rule, e.g. ``src/main/php/PHPMD/Rule/Naming/LongVariable.php``
+- Cover cases for the new property in rule test, e.g. ``src/test/php/PHPMD/Rule/Naming/LongVariableTest.php``
+-- Cover the case when the new property is not set and the rule *should not* apply
+-- Cover the case when the new property is not set and the rule *should* apply
+-- Cover case when the new property is set and the rule *should not* apply
+-- Cover case when the new property is set and the rule *should* apply
+-- Cover edge cases of th new property, if any
```
Comment (tvbeek) [inline .github/PULL_REQUEST_TEMPLATE.md:35]: missing the e for the

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ravage84) [reply .github/PULL_REQUEST_TEMPLATE.md]: @tvbeek thanks, feel free to fix such small issues yourself to keep the time-to-merge small.
